js/app.js

```diff
@@ -4,10 +4,69 @@
 YoastSEO = ( "undefined" === typeof YoastSEO ) ? {} : YoastSEO;
 
```

Comment:
Please change to typeof YoastSEO === "undefined" since that notation is used everywhere in the analyze